modules/ai/assets/js/editor/components/usage-limit-alert.js

```diff
@@ -13,7 +13,13 @@ const CREDITS_80_USAGE_TITLE = getUsageTitle( '80%' );
 const CREDITS_75_USAGE_TITLE = getUsageTitle( '75%' );
 
 const DESCRIPTION_SUBSCRIPTION = __( 'Get maximum access.', 'elementor' );
-const DESCRIPTION_NO_SUBSCRIPTION = __( 'Upgrade now to keep using this feature. You still have credits for other AI features (Text, Code, Images, Containers, etc.)', 'elementor' );
+const FEATURES = [ 'Text', 'Images', 'Code', 'Containers' ];
```

Comment:
Translate?
`{ text: __( 'Text', 'elementor' ), ...

Reply:
@matipojo  Do we need to add translation here if it will later be feed to this string L21?

Reply:
Where will it be translated?